_posts/2019-06-20-june-meetup.md

```diff
@@ -0,0 +1,33 @@
+# June 20th
+
+Hello everyone 😄,
+It is time for the June edition of CopenhagenJS. We are being hosted by Visma.
+
+    Schedule:
+    17:00 Doors open
```

Comment:
Why are some of them indented differently? :)

Reply:
Honestly I dunno, it's from your previous template. But from the view in the editor, it does make it a different level/block. After rendered, the font also looked different

Reply:
It is intentional that it is a quote, so that it uses monospace, but it is this I talk about:

![image](https://user-images.githubusercontent.com/1126497/58771232-5bf60200-85b3-11e9-92f4-38597b285b27.png)

but looking at the sourcecode it is perfect, don't know why it is wrong in the browser

![image](https://user-images.githubusercontent.com/1126497/58771261-73cd8600-85b3-11e9-9543-b76772c4595a.png)

Reply:
@zoeyzou Sorry, it was a chrome extension that made this error, and I didn't think about that :/ I disabled it now